lib/models/project.js

```diff
@@ -496,6 +496,7 @@ class Project {
       return;
     }
     this._addonsInitialized = true;
+    this._didDiscoverAddons = false;
```

Comment:
I assume this was added to fix the `ember install` error?
I made https://github.com/ember-cli/ember-cli/pull/10014 as well in an attempt to resolve that.
I'm not entirely sure which fix is the right one, but it is probably better to have a separate PR and changelog entry for this.

Reply:
It addresses the issue I described in 1. in the PR description. Let me check if your fix does also fix the issue for me, then I'll revert this and wait for your change to get merged first...

Reply:
Yeah, just reran the tests with your change, and they were passing. So I'll just discard my change and rebase when your PR has been merged!

Reply:
Good to know, thanks for confirming!